### ISSUE TITLE
When the duration is more than 2 minutes, trail job status is failed

### ISSUE THREAD

**Issue by kuoorczp** [microsoft/nni#1929]
nnimanager.log:
[2020-1-7 11:48:34] DEBUG [ 'getTrialJobsByReplayEvents done' ]
[2020-1-7 11:48:34] DEBUG [ 'ipcInterface command type: [PI], content:[]' ]
[2020-1-7 11:48:35] DEBUG [ 'trialJob status update: M8agZ, SUCCEEDED' ]
[2020-1-7 11:48:35] INFO [ 'Trial job M8agZ status changed from RUNNING to SUCCEEDED' ]
[2020-1-7 11:48:35] DEBUG [ 'storeTrialJobEvent: event: SUCCEEDED, data: undefined, jobDetail: {"id":"M8agZ","status":"SUCCEEDED","submitTime":1578368836390,"workingDirectory":"C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\M8agZ","form":{"sequenceId":1,"hyperParameters":{"value":"{\\"parameter_id\\":null,\\"parameter_source\\":\\"customized\\",\\"parameters\\":{\\"num_leaves\\":146,\\"learning_rate\\":0.5,\\"bagging_fraction\\":0.9440494376965134,\\"bagging_freq\\":1}}","index":0}},"url":"file://localhost:C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\M8agZ","gpuIndices":[],"startTime":1578368840240,"pid":18936,"endTime":1578368912082}' ]
[2020-1-7 11:48:35] DEBUG [ 'ipcInterface command type: [EN], content:[{"trial_job_id":"M8agZ","event":"SUCCEEDED","hyper_params":"{\\"parameter_id\\":null,\\"parameter_source\\":\\"customized\\",\\"parameters\\":{\\"num_leaves\\":146,\\"learning_rate\\":0.5,\\"bagging_fraction\\":0.9440494376965134,\\"bagging_freq\\":1}}"}]' ]
[2020-1-7 11:48:35] DEBUG [ 'ipcInterface command type: [GE], content:[1]' ]
[2020-1-7 11:48:35] INFO [ 'submitTrialJob: form: {"sequenceId":2,"hyperParameters":{"value":"{\\"parameter_id\\": 1, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"num_leaves\\": 82, \\"learning_rate\\": 0.02, \\"bagging_fraction\\": 0.9081849001346404, \\"bagging_freq\\": 2}, \\"parameter_index\\": 0}","index":0}}' ]
[2020-1-7 11:48:35] DEBUG [ 'submitTrialJob: return: {"id":"SyF9n","status":"WAITING","submitTime":1578368915468,"workingDirectory":"C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\SyF9n","form":{"sequenceId":2,"hyperParameters":{"value":"{\\"parameter_id\\": 1, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"num_leaves\\": 82, \\"learning_rate\\": 0.02, \\"bagging_fraction\\": 0.9081849001346404, \\"bagging_freq\\": 2}, \\"parameter_index\\": 0}","index":0}},"url":"file://localhost:C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\SyF9n","gpuIndices":[]} ' ]
[2020-1-7 11:48:35] DEBUG [ 'storeTrialJobEvent: event: WAITING, data: {"parameter_id": 1, "parameter_source": "algorithm", "parameters": {"num_leaves": 82, "learning_rate": 0.02, "bagging_fraction": 0.9081849001346404, "bagging_freq": 2}, "parameter_index": 0}, jobDetail: {"id":"SyF9n","status":"WAITING","submitTime":1578368915468,"workingDirectory":"C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\SyF9n","form":{"sequenceId":2,"hyperParameters":{"value":"{\\"parameter_id\\": 1, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"num_leaves\\": 82, \\"learning_rate\\": 0.02, \\"bagging_fraction\\": 0.9081849001346404, \\"bagging_freq\\": 2}, \\"parameter_index\\": 0}","index":0}},"url":"file://localhost:C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\SyF9n","gpuIndices":[]}' ]
[2020-1-7 11:48:35] INFO [ 'NNIManager received command from dispatcher: TR, {"parameter_id": 2, "parameter_source": "algorithm", "parameters": {"num_leaves": 131, "learning_rate": 0.05, "bagging_fraction": 0.8963267191943094, "bagging_freq": 4}, "parameter_index": 0}' ]
[2020-1-7 11:48:39] DEBUG [ 'ipcInterface command type: [PI], content:[]' ]
[2020-1-7 11:48:44] DEBUG [ 'GET: /experiment: body:\n{}' ]
[2020-1-7 11:48:44] DEBUG [ 'GET: /check-status: body:\n{}' ]
[2020-1-7 11:48:44] DEBUG [ 'GET: /trial-jobs: body:\n{}' ]

....

[2020-1-7 11:54:15] DEBUG [ 'trialJob status update: SyF9n, FAILED' ]
[2020-1-7 11:54:15] INFO [ 'Trial job SyF9n status changed from RUNNING to FAILED' ]
[2020-1-7 11:54:15] DEBUG [ 'storeTrialJobEvent: event: FAILED, data: undefined, jobDetail: {"id":"SyF9n","status":"FAILED","submitTime":1578368915468,"workingDirectory":"C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\SyF9n","form":{"sequenceId":2,"hyperParameters":{"value":"{\\"parameter_id\\": 1, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"num_leaves\\": 82, \\"learning_rate\\": 0.02, \\"bagging_fraction\\": 0.9081849001346404, \\"bagging_freq\\": 2}, \\"parameter_index\\": 0}","index":0}},"url":"file://localhost:C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\SyF9n","gpuIndices":[],"startTime":1578368920999,"pid":10552,"endTime":1578369255892}' ]
[2020-1-7 11:54:15] DEBUG [ 'ipcInterface command type: [EN], content:[{"trial_job_id":"SyF9n","event":"FAILED","hyper_params":"{\\"parameter_id\\": 1, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"num_leaves\\": 82, \\"learning_rate\\": 0.02, \\"bagging_fraction\\": 0.9081849001346404, \\"bagging_freq\\": 2}, \\"parameter_index\\": 0}"}]' ]
[2020-1-7 11:54:15] DEBUG [ 'ipcInterface command type: [GE], content:[1]' ]
[2020-1-7 11:54:15] INFO [ 'submitTrialJob: form: {"sequenceId":3,"hyperParameters":{"value":"{\\"parameter_id\\": 2, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"num_leaves\\": 131, \\"learning_rate\\": 0.05, \\"bagging_fraction\\": 0.8963267191943094, \\"bagging_freq\\": 4}, \\"parameter_index\\": 0}","index":0}}' ]
[2020-1-7 11:54:15] DEBUG [ 'submitTrialJob: return: {"id":"PACsZ","status":"WAITING","submitTime":1578369255898,"workingDirectory":"C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\PACsZ","form":{"sequenceId":3,"hyperParameters":{"value":"{\\"parameter_id\\": 2, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"num_leaves\\": 131, \\"learning_rate\\": 0.05, \\"bagging_fraction\\": 0.8963267191943094, \\"bagging_freq\\": 4}, \\"parameter_index\\": 0}","index":0}},"url":"file://localhost:C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\PACsZ","gpuIndices":[]} ' ]
[2020-1-7 11:54:15] DEBUG [ 'storeTrialJobEvent: event: WAITING, data: {"parameter_id": 2, "parameter_source": "algorithm", "parameters": {"num_leaves": 131, "learning_rate": 0.05, "bagging_fraction": 0.8963267191943094, "bagging_freq": 4}, "parameter_index": 0}, jobDetail: {"id":"PACsZ","status":"WAITING","submitTime":1578369255898,"workingDirectory":"C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\PACsZ","form":{"sequenceId":3,"hyperParameters":{"value":"{\\"parameter_id\\": 2, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"num_leaves\\": 131, \\"learning_rate\\": 0.05, \\"bagging_fraction\\": 0.8963267191943094, \\"bagging_freq\\": 4}, \\"parameter_index\\": 0}","index":0}},"url":"file://localhost:C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\PACsZ","gpuIndices":[]}' ]
[2020-1-7 11:54:15] INFO [ 'NNIManager received command from dispatcher: TR, {"parameter_id": 3, "parameter_source": "algorithm", "parameters": {"num_leaves": 96, "learning_rate": 0.02, "bagging_fraction": 0.9829264300010454, "bagging_freq": 1}, "parameter_index": 0}' ]
[2020-1-7 11:54:16] DEBUG [ 'ipcInterface command type: [PI], content:[]' ]
[2020-1-7 11:54:21] INFO [ 'Trial job PACsZ status changed from WAITING to RUNNING' ]
[2020-1-7 11:54:21] DEBUG [ 'storeTrialJobEvent: event: RUNNING, data: undefined, jobDetail: {"id":"PACsZ","status":"RUNNING","submitTime":1578369255898,"workingDirectory":"C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\PACsZ","form":{"sequenceId":3,"hyperParameters":{"value":"{\\"parameter_id\\": 2, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"num_leaves\\": 131, \\"learning_rate\\": 0.05, \\"bagging_fraction\\": 0.8963267191943094, \\"bagging_freq\\": 4}, \\"parameter_index\\": 0}","index":0}},"url":"file://localhost:C:\\\\Users\\\\Administrator\\\\nni\\\\experiments\\\\zKzPCmLx\\\\trials\\\\PACsZ","gpuIndices":[],"startTime":1578369258592,"pid":18028}' ]
[2020-1-7 11:54:21] DEBUG [ 'ipcInterface command type: [PI], content:[]' ]
[2020-1-7 11:54:22] DEBUG [ 'GET: /experiment: body:\n{}' ]
[2020-1-7 11:54:22] DEBUG [ 'GET: /check-status: body:\n{}' ]
[2020-1-7 11:54:22] DEBUG [ 'GET: /trial-jobs: body:\n{}' ]
[2020-1-7 11:54:22] DEBUG [ 'GET: /metric-data: body:\n{}' ]

**Comment by SparkSnail** [comment]
Hi @kuoorczp, do you have stderr file in your log path?

**Comment by kuoorczp** [comment]
stderr 's size is 0

**Comment by SparkSnail** [comment]
There is a `.nni` folder in the log path, theoretically there are two files, containing `metric` and `state`. Does your `.nni` folder have `state` file?

**Comment by kuoorczp** [comment]
state:
0 1578379106543

metrics:
ME000103{"parameter_id": 0, "trial_job_id": "q51NP", "type": "PERIODICAL", "sequence": 0, "value": 0.76006044}
ME000098{"parameter_id": 0, "trial_job_id": "q51NP", "type": "FINAL", "sequence": 0, "value": 0.76006044}

**Comment by kuoorczp** [comment]
![nni](https://user-images.githubusercontent.com/59590811/71875571-99c99300-315f-11ea-8f1e-65c2380838a3.png)

**Comment by SparkSnail** [comment]
Hi @kuoorczp,
Seems the log information is insuffiient, could you please try using
```
  git clone -b dev-powershellSupport https://github.com/SparkSnail/nni.git
  cd nni
  powershell -ExecutionPolicy Bypass -file install.ps1
```
to install NNI from source code, and use
```
nnictl create --config {config_file} --debug
```
to start an experiment.
This version of code have more debug information.

**Comment by kuoorczp** [comment]
> Hi @kuoorczp,
> Seems the log information is insuffiient, could you please try using
> 
> ```
>   git clone -b dev-powershellSupport https://github.com/SparkSnail/nni.git
>   cd nni
>   powershell -ExecutionPolicy Bypass -file install.ps1
> ```
> 
> to install NNI from source code, and use
> 
> ```
> nnictl create --config {config_file} --debug
> ```
> 
> to start an experiment.
> This version of code have more debug information.

ok.i'll try

**Comment by scarlett2018** [comment]
> > Hi @kuoorczp,
> > Seems the log information is insuffiient, could you please try using
> > ```
> >   git clone -b dev-powershellSupport https://github.com/SparkSnail/nni.git
> >   cd nni
> >   powershell -ExecutionPolicy Bypass -file install.ps1
> > ```
> > 
> > 
> > to install NNI from source code, and use
> > ```
> > nnictl create --config {config_file} --debug
> > ```
> > 
> > 
> > to start an experiment.
> > This version of code have more debug information.
> 
> ok.i'll try

@kuoorczp how things going with the problem? are you able to walk around it?

**Comment by kvartet** [comment]
I’m closing this issue as it has no updates from customer for 3 months, please feel free to reopen if you are still seeing it an active issue.
